<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>791</width>
      <height>571</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTextEdit" name="textview"/>
     </item>
     <item>
      <widget class="QPushButton" name="bt2">
       <property name="text">
        <string>单句翻译</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bt1">
       <property name="text">
        <string>转换</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="textBrowser"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>789</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="qb_sq">
       <property name="text">
        <string>QB单引号</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="qb_dq">
       <property name="text">
        <string>QB双引号</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="esx_sq">
       <property name="text">
        <string>ESX单引号</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="esx_dq">
       <property name="text">
        <string>ESX双引号</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>825</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
